src/tests/ftest/control/daos_control_config.yaml

```diff
@@ -33,7 +33,7 @@ control_config_val: !mux
     config_val:
       - "name"
       - "! @#$%^&*()_+{}|:<>?-=[];',./"
-      - "PASS"
+      - "FAIL"
```

Comment:
We should expand the coverage to cover special characters and over length, the reason this now fails is because there is a new length requirement introduced by 0b5cdc07a409c525b03dd632626d4fd8753982c0 which limits to 15 chars.

I suggest reducing the value in this test case to 15 chars (which should pass even with special characters) and leaving the expectation as "PASS". Then add another `name_links_exceeded` test case with the original value which should result in a failure.

Reply:
I agree that would be better.

Reply:
> I agree that would be better.

@tanabarr pushed the commit ce831918db33ef78ba682f06a50fbcabba7949b1 to update the unit tests in this way 😸